pwiz_tools/Skyline/FileUI/PeptideSearch/ImportPeptideSearchDlg.cs

```diff
@@ -526,20 +559,68 @@ private void NextPage()
                     }
                     return;
             }
+            case Pages.dda_search_settings_page:
+                    //todo load and start search
+                    bool valid = SearchSettingsControl.SaveAllSettings();
+                    if (!valid) return;
+                    ImportPeptideSearch.SearchEngine.SetEnzyme(Document.Settings.PeptideSettings.Enzyme, Document.Settings.PeptideSettings.DigestSettings.MaxMissedCleavages);
+                    ImportPeptideSearch.SearchEngine.SetSpectrumFiles(ImportPeptideSearch.SearchFilenames);
+                    ImportPeptideSearch.SearchEngine.SetFastaFiles(ImportFastaControl.FastaFile);
+                    SearchControl.OnSearchFinished += SearchControl_OnSearchFinished;
+                    btnNext.Enabled = false;
+                    btnCancel.Enabled = false;
+                    btnBack.Enabled = false;
+                    SearchControl.RunSearch();
+                    break;
+                case Pages.dda_search_page: // this is really the last page
+                                            //todo check if search was correct
+                                            //todo calculate library
+                    var eCancel2 = new CancelEventArgs();
+                    var origName = BuildPepSearchLibControl.SearchFilenames[0];
+                
+                    var outFilePath = Path.GetFileNameWithoutExtension(origName) + ".mzid";
+                    BuildPepSearchLibControl.SearchFilenames[0] = Path.Combine(Path.GetDirectoryName(BuildPepSearchLibControl.SearchFilenames[0]), outFilePath);
+                    BuildPepSearchLibControl.PerformDDASearch = false;
+                    ImportPeptideSearch.SearchFilenames[0] = BuildPepSearchLibControl.SearchFilenames[0];
+                    ImportPeptideSearch.SpectrumSourceFiles.Clear();
+                    ImportPeptideSearch.SpectrumSourceFiles.Add(BuildPepSearchLibControl.SearchFilenames[0], new ImportPeptideSearch.FoundResultsFilePossibilities(origName) { ExactMatch = origName });
+                    if (!BuildPeptideSearchLibrary(eCancel2)) {
+                      // Page shows error
+                      if (eCancel2.Cancel)
+                        return;
+                      CloseWizard(DialogResult.Cancel);
+                    }
+                    //fasta file necessary
+                    //load proteins after search
+                    if (!ImportFastaControl.ImportFasta(ImportPeptideSearch.IrtStandard))
+                      return;
+
+                    WizardFinish();
+                    return;
+
+            }
 
             var newPage = CurrentPage + 1;
             while (_pagesToSkip.Contains(newPage))
                 ++newPage;
 
             // Skip import FASTA if user filters for document peptides
-            if (newPage > Pages.import_fasta_page)
+            if (newPage > Pages.import_fasta_page && !ImportPeptideSearch.IsDDASearch)

```

Comment:
Unless we are going to be writing out a FASTA file if the user pastes in FASTA text, we should make the FASTA tab only accept a FASTA file. I.e. replace the big textbox with a filepath box.